### ISSUE TITLE
Inconsistent invoice dates in the New subscription email

### ISSUE THREAD

**Issue by madalincm** [mozilla/subhub#309]
#### Affected versions:
- Firefox Release 69.0.1
- Firefox Beta 70.0b9
- Firefox Nightly 71.0a1

#### Affected Platforms:
- Windows 10 (64-bit)
- MacOS 10.14.5
- Linux Ubuntu 16.04 (64-bit)

#### Precondition
User is logged in in the Guardian VPN homepage and he does not have an active subscription.
https://stage.guardian.nonprod.cloudops.mozgcp.net/vpn

#### Steps to reproduce
1. Click on the "Subscribe for just $9.99/mo" blue button
2. Enter valid valid info and complete the subscription process
3. Observe the New Subscription email that was sent to the user

#### Expected result:
In the email the next invoice date is tomorrow (10/10.2019)

#### Actual result:
In the email the current invoice is having the present day date (10/9/2019 ) and the next invoice date is 2 days from now (10/11/2019)
Please view this screenshot containing the new subscription email:
![newSubEmail](https://user-images.githubusercontent.com/15685441/66500488-48f7bf00-eaca-11e9-94bf-f5be0c2d77a7.png)

Also in the Subscriptions page the date for the next payment is tommrow (10/10/2019) so I would expect the invoice to be sent in the same date.
Please view this screenshot from the Subscription page: 
![image](https://user-images.githubusercontent.com/15685441/66500363-0df58b80-eaca-11e9-9b5b-11b96bab1ff7.png)

**Comment by andrewmozilla** [comment]
Looking in SF sandbox:  
Billing Cycle StartDate: 10/10
Invoice Number: 734514FA-0002
Billing Cycle End Date: 10/11/2019

No solution yet. Still digging. 

Link to record in SF: https://mozillasalescloud--advo.lightning.force.com/lightning/r/Opportunity/006R000000UvkoLIAR/view

<img width="1024" alt="Screen Shot 2019-10-11 at 9 12 02 AM" src="https://user-images.githubusercontent.com/54546925/66667192-54043980-ec07-11e9-8f14-7684bd96314b.png">

**Comment by desiree-mofo** [comment]
Solution was identified to move to using Invoice date as the billing period start date and next invoice date as billing period end date. Email templates updated as well as documentation and the JIRA ticket related to this GitHub ticket.

**Comment by madalincm** [comment]
I have verified this in stage environment and the dates look good at this moment.  I've checked with both recurrent payments and new subscription emails. 

![recurrentOk](https://user-images.githubusercontent.com/15685441/66924979-45b58380-f034-11e9-8e4a-5ed009e0b3f7.png)

![newSubOk](https://user-images.githubusercontent.com/15685441/66925072-73023180-f034-11e9-8828-3992e9efd88b.png)

Marking bug as verified fixed.
